lib/axiom/fs/base/execute_context.js

```diff
@@ -213,6 +205,27 @@ ExecuteContext.prototype.setStdio = function(stdio) {
 };
 
 /**
+ * @return {!Stdio}
+ */
+ExecuteContext.prototype.getStdio = function() {
+  return new Stdio(
+      this.stdin,
+      this.stdout,
+      this.stderr,
+      this.signal,
+      this.ttyin,
+      this.ttyout
+  );
+};
+
+/**
+ * @return {!NestedStdio}
+ */
+ExecuteContext.prototype.createNestedStdio = function() {
+  return new NestedStdio(this.getStdio());
```

Comment:
This does not break anything right now, but it will soon, because stdio is meant to be a container of any number of streams. So, the NestedStdio constructor needs to have the original stdio (with all the additional streams) to do its work of creating the "wrapper" streams for all the streams (not just the predefined ones).